Dockerfile

```diff
@@ -39,10 +39,14 @@ RUN \
   rm -rf /var/cache/oracle-jdk8-installer
 ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
 
+#Devices get mounted to group 16, need to define that guid as a group to allow openhab user to be added
+RUN groupadd -g 16 dockdev
```

Comment:
Please add the groupadd command to [Line 43](https://github.com/openhab/openhab-docker/blob/master/Dockerfile#L43) as each RUN command produces a new layer.